src/components/ui/Breadcrumb/breadcrumb.scss

```diff
@@ -41,21 +66,94 @@
 
     [data-breadcrumb-divider] {
       height: var(--fs-spacing-3);
-      margin: 0 var(--fs-spacing-2) 0 var(--fs-spacing-1);
+      margin: 0 var(--fs-spacing-1) 0 var(--fs-spacing-1);
       border-left: var(--fs-border-width-default) solid var(--fs-border-color-light);
     }
 
     [data-store-link] {
-      display: inline;
       padding: 0;
 
       &:visited {
         color: var(--fs-color-link);
       }
     }
 
+    [data-store-dropdown-button] {
+      display: inline;
+      margin: calc(var(--fs-spacing-0) / 4);
+      padding: 0;
+      padding: 0 calc(var(--fs-spacing-0) / 2);
+      color: var(--fs-color-link);
+      letter-spacing: var(--fs-spacing-0);
+      background-color: #0000;
+      border-width: 0;
+      border-radius: var(--fs-spacing-0);
+      cursor: pointer;
+
+      &:focus, &:focus-visible {
+        @include focus-ring;
+      }
+
+      span {
+        display: inline-block;
+        font-weight: bold;
+        font-size: var(--fs-text-size-3);
+        transform: translate(rem(1px), rem(-6px));
+      }
+    }
+
     [data-breadcrumb-item="current"] {
       color: var(--fs-color-text-light);
     }
   }
 }
+
+[data-store-dropdown-menu] {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  max-width: 50%;
+  margin-top: var(--fs-spacing-0);
+  overflow: hidden;
+  background: var(--fs-color-tertiary-bkg);
+  border-radius: var(--fs-border-radius-default);
+
+  /* Drop Shadow */
```

Comment:
Maybe we're fine removing this comment as the `box-shadow` below is a good indication already.